src/data_scripts/read_data.py

```diff
@@ -0,0 +1,35 @@
+from google.cloud import storage
```

Comment:
You should remove this file if not it will cause merge conflicts with your previous PR

Reply:
Done. Yeah that makes sense.